ofctrl/fgraphFlow.go

```diff
@@ -450,8 +451,17 @@ func (self *Flow) xlateMatch() openflow15.Match {
 	if self.Match.XxRegs != nil {
```

Comment:
I wonder why this case (xxreg) is so different from the one above (reg). In the above case, I don't even see a reference to `reg.Range`.

Reply:
reg.Range is used in "merge" function which is called if the caller uses more than one match fields in the same reg. In ofnet, we would merge the these fields into a final data/mask, and then send to OVS.